### PR TITLE
Repo create

### PR DESCRIPTION
**Pull request by jdextercsc**
Started at the end of base role so not. 

[root@kragle2 ansible]# ansible-playbook main.yml --start-at-task="clone repos from git" -vv --step

PLAY [Setup kragle] ***********************************************************

GATHERING FACTS ***************************************************************
<localhost> REMOTE_MODULE setup
ok: [localhost]
Perform task: base | clone repos from git (y/n/c): n
Perform task: createrepo | create repo directory for httpd (y/n/c): y

Perform task: createrepo | create repo directory for httpd (y/n/c):  **********
<localhost> REMOTE_MODULE file state=directory path=/var/www/html/kragle-offline
ok: [localhost] => {"changed": false, "gid": 0, "group": "root", "mode": "0755", "owner": "root", "path": "/var/www/html/kragle-offline", "secontext": "unconfined_u:object_r:httpd_sys_content_t:s0", "size": 49152, "state": "directory", "uid": 0}
Perform task: createrepo | copy repo files to http directory (y/n/c): n
Perform task: createrepo | createrepo (y/n/c): y

Perform task: createrepo | createrepo (y/n/c):  *******************************
<localhost> REMOTE_MODULE command createrepo "/var/www/html/kragle-offline"
changed: [localhost] => {"changed": true, "cmd": ["createrepo", "/var/www/html/kragle-offline"], "delta": "0:00:05.914658", "end": "2015-09-28 16:42:28.555692", "rc": 0, "start": "2015-09-28 16:42:22.641034", "stderr": "", "stdout": "Spawning worker 0 with 220 pkgs\nSpawning worker 1 with 220 pkgs\nSpawning worker 2 with 219 pkgs\nSpawning worker 3 with 219 pkgs\nWorkers Finished\nSaving Primary metadata\nSaving file lists metadata\nSaving other metadata\nGenerating sqlite DBs\nSqlite DBs complete", "warnings": []}
Perform task: createrepo | repofile to provide to node (y/n/c): y

Perform task: createrepo | repofile to provide to node (y/n/c):  **************
changed: [localhost] => {"changed": true, "checksum": "2bd009434704a1885684936eb026fc5468a4a212", "dest": "/var/www/html/offline.repo", "gid": 0, "group": "root", "md5sum": "80392bba93a1df2b16d47b0b2b2c4d46", "mode": "0644", "owner": "root", "secontext": "system_u:object_r:httpd_sys_content_t:s0", "size": 138, "src": "/root/.ansible/tmp/ansible-tmp-1443472950.59-152144187679932/source", "state": "file", "uid": 0}

PLAY RECAP ********************************************************************
localhost                  : ok=4    changed=2    unreachable=0    failed=0

### CODE REVIEW

**Comment by mtnbikenc** [comment]
Looks like `httpd` is disabled and not started by default.  It should probably be started and enabled.

After `yum install -y httpd`
```
[root@kragle ~]# systemctl status httpd
httpd.service - The Apache HTTP Server
   Loaded: loaded (/usr/lib/systemd/system/httpd.service; disabled)
   Active: inactive (dead)
```

File: ansible/roles/createrepo/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+- name: repofile to provide to node
+  template:
+    src: repo_template.repo.j2
+    dest: "/var/www/html/offline.repo"
+
+- name: start httpd
+  service:
+    name: httpd
+    state: restarted
+
```
Comment (mtnbikenc) [inline ansible/roles/createrepo/tasks/main.yml]: This should include `enabled: yes`

**Comment by mtnbikenc** [comment]
Code review :+1:

**Comment by jcpowermac** [comment]
Code review :+1:

**Pull request merged**